test/models/game_test.rb

```diff
@@ -1,33 +1,32 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
- test "Initial board position" do
-   
-    game = Game.create(:user_id => 1)
-    
-       
-    expected = [["00", "10", "20", "30", "40", "50", "60", "70"],
-                ["01", "11", "21", "31", "41", "51", "61", "71"],
-                [nil, nil, nil, nil,nil,nil,nil,nil],
-                [nil, nil, nil, nil,nil,nil,nil,nil],
-                [nil, nil, nil, nil,nil,nil,nil,nil],
-                [nil, nil, nil, nil,nil,nil,nil,nil],
-                ["06", "16", "26", "36", "46", "56", "66", "76"],
-                ["07", "17", "27", "37", "47", "57", "67", "77"]].flatten
+ 
+  game = Game.create(:user_id => 1)
```

Comment:
Same here, you don't want to create items in your database outside a "test" block; it causes weirdness to happen.